devel/libatomic_ops/Portfile

```diff
@@ -8,10 +8,10 @@ name                libatomic_ops
 subport             libatomic_ops-devel {}
 
 if {${subport} eq ${name}} {
-    github.setup    ivmai libatomic_ops 7.6.2 v
+    github.setup    ivmai libatomic_ops 7.6.4 v
 
-    checksums       rmd160  994eee4df05d0c3eed92d9946e24f35642a32807 \
-                    sha256  6b378569f52f97cf26c3ab9d5098ae9c418361e26c13aac6d1527f7fd76ddfe2
+    checksums       rmd160  74a4f7657d62da095d09f50e8291928f5ec8a43b \
```

Comment:
Sorry. I think I've provided wrong checksums. How to compute them correctly? (I've computed by "wget https://github.com/ivmai/libatomic_ops/releases/download/v7.6.4/libatomic_ops-7.6.4.tar.gz && openssl dgst -rmd160 libatomic_ops-7.6.4.tar.gz")

Reply:
You can compute them by trying "port -v checksum" at the command line. It will give you the correct ones.